src/Microsoft.OData.Extensions.Client/Microsoft.OData.Extensions.Client.Nightly.nuspec

```diff
@@ -3,31 +3,34 @@
     <metadata>
         <id>Microsoft.OData.Extensions.Client</id>
         <title>Microsoft Extensions OData Client</title>
-        <version>$VersionFullSemantic$-Nightly$NightlyBuildVersion$</version>
+        <version>$version$-Nightly$NightlyBuildVersion$</version>
         <authors>OData (.NET Foundation)</authors>
         <copyright>&#169; .NET Foundation and Contributors. All rights reserved.</copyright>
         <description>This package contains the OData Client factory, with built-in handler to hook up with Http client factory.</description>
         <summary>The OData Client factory is a pattern for configuring and retrieving named OData client in a composable way. The OData Client factory provides extensibility to plug in OData client handler that address cross-cutting concerns. The default OData client factory provides built-in handler to hook up with Http client factory.
-   Commonly used types:
-     Microsoft.OData.Extensions.Client.IODataClientFactory
-     Microsoft.OData.Extensions.Client.IODataClientBuilder
-     Microsoft.OData.Extensions.Client.ODataClientBuilderExtensions
-     Microsoft.OData.Extensions.Client.ODataClientServiceCollectionExtensions.</summary>
+        Commonly used types:
+           Microsoft.OData.Extensions.Client.IODataClientFactory
+           Microsoft.OData.Extensions.Client.IODataClientBuilder
+           Microsoft.OData.Extensions.Client.ODataClientBuilderExtensions
+           Microsoft.OData.Extensions.Client.ODataClientServiceCollectionExtensions.
+        </summary>
         <language>en-US</language>
         <projectUrl>http://github.com/OData/Extensions</projectUrl>
         <license type="expression">MIT</license>
         <requireLicenseAcceptance>true</requireLicenseAcceptance>
         <tags>Microsoft Extensions ODataClient AspNetCore</tags>
         <dependencies>
+          <group targetFramework=".NETStandard2.0">
             <dependency id="Microsoft.Extensions.DependencyInjection" version="$ExtensionsDependencyInjectionDependency$" />
             <dependency id="Microsoft.Extensions.Options" version="$ExtensionsOptionsDependency$" />
             <dependency id="Microsoft.Extensions.Http" version="$ExtensionsHttpDependency$" />
-            <dependency id="Microsoft.OData.Extensions.Client.Abstractions" version="[$VersionFullSemantic$-Nightly$NightlyBuildVersion$]" />
+            <dependency id="Microsoft.OData.Extensions.Client.Abstractions" version="$VersionSemanticsClientAbstractions$" />
```

Comment:
>$VersionSemanticsClientAbstractions$ [](start = 85, length = 36)

In nightly, we should target to the nightly version?

Reply:
Since both were being created in the same run it would probably be better to target the nightly here